bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+from bot.constants import Tokens
+
+log = logging.getLogger(__name__)
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        async with ctx.typing():
+            async with self.bot.http_session.get(
+                    'https://api.unsplash.com/photos/random',
+                    params={"query": "earth", "client_id": Tokens.unsplash_access_key}) as r:
+                jsondata = await r.json()
+                linksdata = jsondata.get("urls")
+                embedlink = linksdata.get("regular")
+                downloadlinksdata = jsondata.get("links")
+                userdata = jsondata.get("user")
+                username = userdata.get("name")
+                userlinks = userdata.get("links")
+                profile = userlinks.get("html")
+            async with self.bot.http_session.get(
+                downloadlinksdata.get("download_location"),
+                    params={"client_id": Tokens.unsplash_access_key}
+            ) as _:
+                pass
```

Comment:
Can we add a comment here saying why are we doing this?

Reply:
This hasn’t being mentioned yet.

Reply:
oh sorry, i'll fix that!

Reply:
fixed as of commit https://github.com/python-discord/sir-lancebot/pull/609/commits/68ff4f3a524e90099e6953bd601321623e789e66